pkg/infra/iac3/resource.go

```diff
@@ -41,9 +41,16 @@ func (tc *TemplatesCompiler) RenderResource(out io.Writer, rid construct.Resourc
 			return err
 		}
 	}
-	err = resTmpl.Template.Execute(out, inputs)
-	if err != nil {
-		return fmt.Errorf("could not render resource %s: %w", rid, err)
+	if r.Imported {
+		err = resTmpl.ImportResource.Execute(out, inputs)
```

Comment:
So this combined with not having a distinguishing "identifier" flag on a property (and just relying on `required`) means that we can't have any default import templates. Fine for now if we only want a handful of resources to be importable, but I think we should consider how to scale this up to be able to import any resources we support.

Reply:
i guess it depends on if one deploy time property is always all it takes to import things. If so we can use the one required deploy time property for a base template, which shouldnt be too hard, but also writing the importFunc in the template should be real quick and simple as well

Reply:
> the one required deploy time property

Is this guaranteed to be singular?

Reply:
it doesnt have to be, i dont know what resource would require multiple, but i could definitely see that happening